archive_upload/handlers/dsmc_handlers.py

```diff
@@ -486,6 +493,9 @@ def post(self, runfolder_archive):
             "archive_description": uniq_id,
             "archive_host": socket.gethostname() }
 
+        if tsm_mock_enabled:
+            response_data["tsm_mock_enabled"] = tsm_mock_enabled
```

Comment:
I'm torn on this. I think that it's a really good idea to give this info to the user, but at the same time it breaks the contract, i.e. we get slightly different data when running in this mode and when not. That might be something the caller isn't expecting so it might actually not work down the line.

How about if we instead add this to a `message` field and make sure to always return a message (empty one if there's nothing to report)?

Reply:
Done